documentation.md

```diff
@@ -34,6 +34,19 @@ title: QuTiP Documentation
 
 ## Previous releases
 
+### Version 4.7
```

Comment:
Version 4.7 is now both in previous releases and latest releases

Reply:
I wanted to keep both versions of the documentation findable so that the list of links to the documentation archived here is complete, but I don't have a much better plan. For the moment I just added an `(archived)` marker. Another option would be to move the archived documentation to a completely separate page.